VK/VK/ClientsGroupsScreen/ClientsGroupsTableViewController.swift

```diff
@@ -96,17 +100,15 @@ final class MyGroupsTableViewController: UITableViewController {
         regCells()
     }
 
-    private func fetchClientsGroups(completion: @escaping () -> ()) {
-        networkService.fetchClientsGroups { [weak self] result in
-            guard let self = self else { return }
-            switch result {
-            case let .success(groupsResponse):
-                let groups = groupsResponse.response.items
-                self.saveData(groups: groups)
-                completion()
-            case let .failure(error):
-                print("\(error)")
-            }
+    private func fetchClientsGroups(completion: () -> ()) {
+        firstly {
+            // TODO: Убрать форс анрап.
```

Comment:
```suggestion
            // TODO: Убрать форс анрап.
```
Я бы рекомендовал отключить линтер до и после этого метода (так он хоть по остальному документу будет работать)

Reply:
Справедливо.